modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -184,6 +187,17 @@ const Renderer = ({
 		PAGE_HEADER_COMPONENT_TYPE[page.headerRenderer] ||
 		Components.PageHeader;
 
+	let hasFieldRequired = false;
+	const visitor = new PagesVisitor(pages);
+	visitor.mapFields((field) => {
+		if (field.required) {
+			hasFieldRequired = true;
+		}
+	});
+	const isAdminPortletNamespace = portletNamespace.includes('Admin');
```

Comment:
@georgel-pop-lr, I believe that we should check for the full namespace, `DDMFormAdmin` (I guess this the full namespace), instead of only `Admin`. Besides, can we move this content (`DDMFormAdmin`) to a constant?

Reply:
@natocesarrego Change done please check. Regards.